### PR TITLE
Change conflict resolution instructions

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

Tell the users to feel free to _append_ to the default merge commit message that `git` creates for them, but insist that they _do not_ completely overwrite it.

## 💭 Motivation and Context

The content the default merge commit message from `git` provides is useful and we should keep it around.

## 🧪 Testing

App pre-commit hooks pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Works for me! ⚙️

File: src/lineage/templates/conflict_template.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ that you must resolve before merging this pull request!
 
     ```console
     git add {{#conflict_file_list}}{{.}} {{/conflict_file_list}}
-    git commit --message="Resolve Lineage conflicts."
+    # You may append to the default merge commit message that git creates for
```
Comment (hillaryj) [inline src/lineage/templates/conflict_template.md]: As someone who copies and pastes the block of commands, I'd recommend moving the directive out of the comments here and probably either deleting it entirely or putting it in the text above like:

> **Note:** Feel free to append to the default merge commit message, but _do not_ overwrite it on the `git commit` command, as the content it provides is useful.

Comment (jsf9k) [reply src/lineage/templates/conflict_template.md]: I'm happy to move it, but the comments shouldn't cause you any difficulties if you copy the whole block wholesale.  Is this a `zsh` thang?

Comment (jsf9k) [reply src/lineage/templates/conflict_template.md]: Addressed in commit 21a3634.

**Review by dav3r — APPROVED** [review]
I'm good with the revised wording. 👍

File: src/lineage/templates/conflict_template.md (hunk 1 above)
Comment (hillaryj) [reply src/lineage/templates/conflict_template.md]: You're right, it wouldn't cause actual technical issues - more of an aesthetic issue I guess.

**Review by hillaryj — APPROVED** [review]
🌮

**Review by felddy — APPROVED** [review]
Good idea.  The default message is useful! 🎃

**Pull request merged**